File: decompiled/3.16.1/be/ehealth/businessconnector/therlink/domain/requests/RevokeTherapeuticLinkRequest.java
package be.ehealth.businessconnector.therlink.domain.requests;

import be.ehealth.businessconnector.therlink.domain.Author;
import be.ehealth.businessconnector.therlink.domain.Proof;
import be.ehealth.businessconnector.therlink.domain.TherapeuticLink;
import be.ehealth.businessconnector.therlink.domain.TherapeuticLinkRequestType;
import org.joda.time.DateTime;

public class RevokeTherapeuticLinkRequest extends TherapeuticLinkRequestType {
   private static final long serialVersionUID = 1L;

   public RevokeTherapeuticLinkRequest(DateTime date, String id, Author author, TherapeuticLink link, Proof... proofs) {
      super(id, date, author, link, proofs);
   }
}

File: src/main/kotlin/org/taktik/connector/business/therlink/domain/requests/RevokeTherapeuticLinkRequest.kt
package org.taktik.connector.business.therlink.domain.requests

import org.taktik.connector.business.therlink.domain.Author
import org.taktik.connector.business.therlink.domain.Proof
import org.taktik.connector.business.therlink.domain.TherapeuticLink
import org.taktik.connector.business.therlink.domain.TherapeuticLinkRequestType
import java.util.Date
import org.joda.time.DateTime

class RevokeTherapeuticLinkRequest(date: DateTime, id: String, author: Author, link: TherapeuticLink?, proofs: List<Proof?> = ArrayList()) : TherapeuticLinkRequestType(date, id, author, link, proofs) {
	@Deprecated("")
	constructor(date: Date, id: String, author: Author, link: TherapeuticLink?, vararg proofs: Proof?) : this(DateTime(date), id, author, link, proofs.toList())
	constructor(date: DateTime, id: String, author: Author, link: TherapeuticLink?, vararg proofs: Proof?) : this(date, id, author, link, proofs.toList())

	companion object {
		private val serialVersionUID = 1L
	}
}
